test/e2e/postgres_restart_test.go

```diff
@@ -15,11 +15,18 @@ import (
 
 func invokeOLS(env *OLSTestEnvironment, secret *corev1.Secret, query string, expected_success bool) {
 	reqBody := []byte(`{"query": "` + query + `"}`)
-	resp, body, err := TestHTTPSQueryEndpoint(env, secret, reqBody)
-	Expect(err).NotTo(HaveOccurred())
-	defer resp.Body.Close()
-	fmt.Println(GinkgoWriter, string(body))
-	Expect(resp.StatusCode == http.StatusOK).To(Equal(expected_success))
+	// Give up after 5 minutes
+	for i := 0; i < 60; i++ {
```

Comment:
we can use `Eventually()` like in these test codes: https://github.com/openshift/lightspeed-operator/blob/main/test/e2e/database_test.go#L211-L220